### DISCUSSION TITLE
Minimum VPS requirements

### DISCUSSION THREAD

**Discussion by markusqw1** [simple-login/app#1789] [Self-hosting question]
Hi, I am thinking about self hosting. The provider I am looking at is 1984.hosting. The VPS plan I am looking at is 1GB RAM
1 CPU 25GB DISK 1TB TRANSFER. Is the Plan good for SimpleLogin or do I need a more powerful plan.

**Comment by Spitfireap** [comment]
Hi, I'm running it on a 1vCPU and 1GB RAM as well (with an old hardware - ddr3-) so no problem I think
